spork/misc.janet

```diff
@@ -121,3 +121,17 @@
       (each child node (dfs child visit-leaf node-before node-after get-children seen))
       (when node-after (node-after node)))
     (visit-leaf data)))
+
+
+(defn randomize-array
+  "Randomizes array using the fisher-yates shuffle, takes an optional random number generator"
+  [arr &opt rng-in]
+  (def rng (if (= rng-in nil) (math/rng (os/cryptorand 8)) rng-in))
```

Comment:
Here I would recommend`default` macro.

Reply:
Ah yes, when I wrote this code, I didn't know about that.
Should be fixed now.